app/controllers/food_access_points_controller.rb

```diff
@@ -46,9 +47,12 @@ def destroy
 
   private 
 
-  def food_access_point_params(*args) 
-    params.require(:food_access_point).permit(*args)
+  def food_access_point_edit_params(fap_params) 
+    fap_params.require(:food_access_point).permit(:name, :address, :town, :state, :zip, :phone_number, :website)
```

Comment:
so you don't actuall need to pass params in here, you can just define the method like
```
  def food_access_point_edit_params 
    params.require(:food_access_point).permit(:name, :address, :town, :state, :zip, :phone_number, :website)
  end
```

and call it like `food_access_point_edit_params`, same with the below method

Reply:
oh woops i see you made this change in the very next commit